docs/superpowers/plans/2026-06-08-claims-immutable-inputs.md

```
@@ -0,0 +1,500 @@
+  it("short-circuits a previously-verified CID without re-hashing", async () => {
+    expect(await verifyCid(HELLO, HELLO_CID)).toBe(true);
+    // second call returns true even if given wrong bytes, because CID is cached-verified
+    expect(await verifyCid(new Uint8Array(), HELLO_CID)).toBe(true);
+  });
+});
+```
+
+- [ ] **Step 3: Print the real CID for the fixture, paste into test**
+
+Run:
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)